packages/example/src/pages/guides/navigation/tabs.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Tabs
+tabs: ['Sidebar', 'Tabs']
+---
+
+<PageDescription>
+
+You have the option of adding tabs to your pages. This is really only recommended if you have many pages with the same tabs. If you just have one page a set of tabs, it might be better for discoverability to make those menu items.
+
+</PageDescription>
+
+## Tabs in YAML
+
+To create a tabbed page, you just need to point the theme to the path of your first tab. This is the structure of the YAML configuration for this page and it's siblings.
+
+```yaml
+- title: Guides
+  pages:
+    - title: Configuration
+      path: /guides/configuration
+    - title: Shadowing
+      path: /guides/shadowing
+    - title: Styling
+      path: /guides/styling
+    - title: Navigation
+      path: /guides/navigation/sidebar
+```
+
+## File structure
+
+Lets check out the directory structure for this page. Notice how you'll add another directory that corresponds with the one in the YAML file.
```

Comment:
This may be way too specific of me,  my bad if it is!
```suggestion
Let's check out the directory structure for this page. Notice how you'll add another directory that corresponds with the one in the YAML file.
```

Reply:
Not at all!